.github/workflows/ci_pytorch.yml

```diff
@@ -1,33 +1,105 @@
-# This is a basic workflow to help you get started with Actions
+# Example workflow to run TPU tests in Github Actions.
 
 name: CI
 
-# Controls when the action will run. Triggers the workflow on push or pull request
-# events but only for the master branch
+# Controls when the action will run.
+#
+# If `pull_requests` is enabled, the workflow will run for every commit
+# pushed to a pending PR.
+#
+# If `schedule` is enabled, the workflow will run at the specified times.
 on:
```

Comment:
Each time a PR is merged into master, a simple TPU test will run in this repo

Reply:
I'm not sure if this is a good idea. Basically all of our PRs come through during hours when TPU capacity is the most limited, so we're going to see a good number of failures caused by stockouts polluting the check history on master.

It may make sense to move this whole example to a separate, otherwise-empty branch and set up the automated on-push trigger on there. Maybe even include some sample code there to better show how this would fit into another repository. It would match the real use-case better if there were separate steps to check out the pending change and check out our templates

Reply:
For now, I've switched to a `schedule` method of running the workflow and running it only once per day. I looked through all our GPU tests and they all use either `'0 6 * * *'` or `'0 8 * * *'`, so I'll use `'0 5 * * *'` since this is a short test. It shouldn't interfere with the existing GPU tests